src/Table.jsx

```diff
@@ -363,6 +368,7 @@ export default class Table extends React.Component {
           onRowDoubleClick={onRowDoubleClick}
           onRowMouseEnter={onRowMouseEnter}
           onRowMouseLeave={onRowMouseLeave}
+          onContextMenu={onContextMenu}
```

Comment:
why not name it `onRowContextMenu`, api follow the same name convention is better